ocp_resources/network_attachment_definition.py

```diff
@@ -211,26 +213,33 @@ def __init__(
         https://docs.openshift.com/container-platform/4.14/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-additional-networks_configuring-additional-network
 
         Args:
-            network_name (str): The name of the network. This field is mandatory when not using yaml
+            network_name (str): The name of the network.
```

Comment:
This should be name of the network or `namespace/network-name`?

Reply:
add that if not sent, the name will be `{self.namespace}/{self.name}`

Reply:
Yes, the name of the network should be defined exactly as you wrote - namespace/nad.name